docs/configure.md

```diff
@@ -0,0 +1,142 @@
+
+```sh
+java -Dotel.service.name=my-service ...
+```
+
+### Configuration file
+
+<!-- ✅ What and why -->
+EDOT Java can be configured using a configuration file.
+
+<!-- ✅ How -->
+Before starting the JVM, create and populate the configuration file and specify where to find it:
+
+```sh
+echo otel.service.name=my-service > my.properties
+java -Dotel.javaagent.configuration-file=my.properties ...
+```
+
+## Configuration options
+
+Because the Elastic Distribution of OpenTelemetry Java is an extension of the [OpenTelemetry Java agent](https://github.com/open-telemetry/opentelemetry-java-instrumentation), it supports both:
+
+* General OpenTelemetry SDK configuration options
+* General OpenTelemetry Java agent configuration options
+* Elastic-specific configuration options that are only available when using EDOT Java
+
+### OpenTelemetry configuration options
+
+EDOT Java supports all configuration options listed in the [OpenTelemetry General SDK Configuration documentation](https://opentelemetry.io/docs/languages/sdk-configuration/general/) and [OpenTelemetry Java agent documentation](https://opentelemetry.io/docs/zero-code/java/agent/configuration/) .
+
+### Elastic distro configuration options
+
+In addition to general OpenTelemetry configuration options, there are two kinds
+of configuration options that are only available in EDOT Java:
+
+* Additional `OTEL_` options that Elastic plans to contribute upstream to the [OpenTelemetry Java agent](https://github.com/open-telemetry/opentelemetry-java-instrumentation), or have been contributed to the
+[Java agent contrib repo](https://github.com/open-telemetry/opentelemetry-java-contrib/) and are included
+in the EDOT but are not yet available in the OpenTelemetry Java agent.
+* `ELASTIC_OTEL_` options that are specific to Elastic and will always live in EDOT Java (in other words, they will _not_ be added upstream).
+
+<!--
+TO DO:
+List config options instead of linking to the README
+-->
+Find a list of configuration options that are only available in EDOT Java in [README](https://github.com/elastic/elastic-otel-java?tab=readme-ov-file#features).
+
+<!-- ✅ List auth methods -->
+## Authentication methods
+
+When sending data to Elastic, there are two ways you can authenticate: using an APM agent key or using a secret token.
+Both EDOT Java and APM Server must be configured with the same secret token for the request to be accepted.
+
+### Use an APM agent key (API key)
+
+<!-- ✅ What and why -->
+[APM agent keys](https://www.elastic.co/guide/en/observability/current/apm-api-key.html) are
+used to authorize requests to an Elastic Observability endpoint.
+APM agent keys are revocable, you can have more than one of them, and
+you can add or remove them without restarting APM Server.
+
+<!-- ✅ How do you authenticate using this method? -->
+To create and manage APM Agent keys in Kibana:
+
+1. Go to **APM Settings**.
+1. Select the **Agent Keys** tab.
```

Comment:
Enumeration where all start with `1.`? Or is there some black magic in the build updating those?

Reply:
It's markdown, markdown generates the numbers

Reply:
TIL, I didn't know of this feature in markdown